src/doom/d_main.h

```diff
@@ -28,6 +28,7 @@
 #ifndef __D_MAIN__
 #define __D_MAIN__
 
+#include "Action.hpp"
```

Comment:
As mentioned in the comment regarding CMakeLists.txt, I'd use `#include "gvizdoom/Action.hpp"` here.